backends-clickhouse/src/main/scala/org/apache/spark/shuffle/CHColumnarShuffleWriter.scala

```diff
@@ -49,7 +49,7 @@ class CHColumnarShuffleWriter[K, V](
   private val subDirsPerLocalDir = blockManager.diskBlockManager.subDirsPerLocalDir
   private val splitSize = GlutenConfig.getConf.maxBatchSize
   private val customizedCompressCodec =
-    GlutenShuffleUtils.getCompressionCodec(conf)
+    GlutenShuffleUtils.getCompressionCodec(conf).toUpperCase(Locale.ROOT)
```

Comment:
Do we need to support both uppercase and lowercase codec config? I would suggest to align with spark config `spark.io.compression.codec`  to only support lower case.

Reply:
I agree. @zzcclp Can you take a look? Using lower case codec string will fail on CH native side.

Reply:
In CH, it only support uppercase codec.

Reply:
So now we are align with Vanilla Spark's config but convert to uppper in Gluten, right? if so it's OK

Reply:
Yes, we're align with spark's config. Here we only convert to uppercase for CH backend.